components/bl-list-component/README.md

```diff
@@ -0,0 +1,55 @@
+# List
+
+List is the component that can be used in Backendless [UI-Builder](https://backendless.com/developers/#ui-builder). It allows you to add a standard list to your application.
+
+<p align="center">
+  <img src="./thumbnail.png" alt="main thumbnail" width="643"/>
+</p>
+
+Additional properties allow customizing the font size, background color, text color, padding and behavior of the component. Refer to the Properties section below.
+
+## Configuration
+
+The configuration can be done in the UI Builder or using the Codeless Logic. You need to select the type of list (ordered/unordered) and specify the data that will be displayed inside the list.
+
+## Properties
+
+| Property                 | Type       | Default value | Logic              | Data Binding | UI Setting | Description                                                        |
+|--------------------------|------------|---------------|--------------------|--------------|------------|-------------------------------------------------------------------|
+| Type                     | *Select*   | `"Unordered"` | Type Logic         | NO           | YES        | Controls the type of list(ul/ol).                                                               |
```

Comment:
Make as [here](https://github.com/Backendless/ui-builder-library/tree/main/components/bl-multiple-bar-chart-component) (Grid Marks property is example)